MapboxNavigation/Constants.swift

```diff
@@ -6,19 +6,14 @@ typealias CongestionSegment = ([CLLocationCoordinate2D], CongestionLevel)
 /**
  Line width base values used at zoom levels.
  */
-public let lineWidthAtZoomLevels: [Int: MGLStyleValue<NSNumber>] = [
+public let MBRoutelineWidthAtZoomLevels: [Int: MGLStyleValue<NSNumber>] = [
```

Comment:
Nit: “route line” is two words. Also, the usual phrasing for a dictionary would be “by zoom level”. So `MBRouteLineWidthByZoomLevel`.